tutorials/modeling_2D.ipynb

```diff
@@ -0,0 +1,320 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(analysis.datasets[\"stacked\"].counts)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(analysis.datasets[\"stacked\"].background_model.map)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(analysis.datasets[\"stacked\"].exposure)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Modelling\n",
+    "\n",
+    "Now, we define a model to be fitted to the dataset. **The important thing to note here is the dummy spectral model - an integrated powerlaw with only free normalisation**. Here, we use its YAML definition to load it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_config = \"\"\"\n",
+    "components:\n",
+    "- name: GC-1\n",
+    "  type: SkyModel\n",
+    "  spatial:\n",
+    "    type: PointSpatialModel\n",
+    "    frame: galactic\n",
+    "    parameters:\n",
+    "    - name: lon_0\n",
+    "      value: 0.02\n",
+    "      unit: deg\n",
+    "    - name: lat_0 \n",
+    "      value: 0.01    \n",
+    "      unit: deg\n",
+    "  spectral:\n",
+    "    type: PowerLaw2SpectralModel\n",
+    "    parameters:\n",
+    "    - name: amplitude      \n",
+    "      value: 1.0e-12\n",
+    "      unit: cm-2 s-1 \n",
+    "    - name: index\n",
+    "      value: 2.0\n",
+    "      unit: ''\n",
+    "      frozen: true\n",
+    "    - name: emin\n",
+    "      value: 0.1\n",
+    "      unit: TeV\n",
+    "      frozen: true\n",
+    "    - name: emax\n",
+    "      value: 10.0\n",
+    "      unit: TeV\n",
+    "      frozen: true \n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "analysis.set_models(model_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will freeze the parameters of the background"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "analysis.datasets[\"stacked\"].background_model.parameters[\"norm\"].frozen = True\n",
+    "analysis.datasets[\"stacked\"].background_model.parameters[\"tilt\"].frozen = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To run the fit\n",
+    "analysis.run_fit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To see the best fit values along with the errors\n",
+    "analysis.fit_result.parameters.to_table()"
```

Comment:
How about showing more here. For instance computing and plotting position error contours for the source position? Does it slow down the notebook a lot?

Reply:
On my machine, adding a call to `analysis.fit.minos_contour` increases the execution time form 20 secs to 33 secs. Perhaps this should better reside on the models notebook?

Reply:
For how many points?

Reply:
The default, 10 points

Reply:
OK. Then leave as is. Just mention the modeling/fitting notebook for now.